tools/tripal/analysis_load_blast.xml

```diff
@@ -1,52 +1,42 @@
-<tool id="load_blast" name="Load Blast results" version="@WRAPPER_VERSION@.0">
+<?xml version="1.0"?>
+<tool id="tripaille.analysis.load_blast" profile="16.04" name="Load Blast results" version="@WRAPPER_VERSION@.0">
     <description>into Tripal</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <expand macro="requirements"/>
     <code file="tripal.py"/>
+    <expand macro="stdio"/>
     <command><![CDATA[
-        tmp_dir=`mktemp -d -p "@DATA_DIR@"`
+        tmp_dir=`mktemp -d -p "@DATA_DIR@"` && chmod a+rx "\${tmp_dir}"
 
         &&
 
-        cp '${results}' "\${tmp_dir}/blast_input.xml"
+        cp '${blast_output}' "\${tmp_dir}/blast_input.xml"
 
         &&
 
-        tripal load_blast
+        @AUTH@
 
-            @ANALYSIS@
-
-            --is-concat
-
-            --blastdb '${blastdb}'
-
-            --blast-parameters '${params}'
+        tripaille analysis load_blast
 
-            --max-hits '${max_hits}'
+            --is_concat
 
-            #if str($query_re):
-                --query-re '${query_re}'
-            #end if
+            --blastdb_id '${blastdb}'
 
-            $query_uniquename
+            --blast_parameters '${params}'
 
-            --query-type '$query_type'
+            @MATCHING@
 
-            @AUTH@
+            @ANALYSIS@
 
             "\${tmp_dir}/blast_input.xml"
-
-            > $output_log
```

Comment:
no outputs?

Reply:
Yep, the only thing we get from tripal are the stdout and stderr of the job, which are printed as is by tripaille and captured by galaxy.
I don't know what I could write in an output file, apart from 'ok'. Ideas welcome if you have some!

Reply:
Even 'ok' would be fine, otherwise the dataset would disappear. I had some tools that did this and it was fantastically annoying. No datasets = no confirmation that the tool ran successfully.